sys/cbor/cbor.c

```diff
@@ -92,6 +96,18 @@
 #endif
 
 #ifndef CBOR_NO_FLOAT
+
+/* pack to force aligned access on ARMv7 (buggy GCC) */
+#pragma GCC diagnostic error "-Wcast-align"
```

Comment:
Quoting yourself in https://github.com/RIOT-OS/RIOT/pull/2189#issuecomment-273108247

> Concerning the pragmas: they should be avoided wherever possible. @Kijewski, any way to get rid of them?

I'm not sure I understand the context of the comment, the pragma and the `packed` attribute, so please provide better context.